pkg/server/commands/check_command.go

```diff
@@ -118,20 +124,14 @@ func (c *CheckQuery) Execute(ctx context.Context, req *openfgav1.CheckRequest) (
 	return resp, resolveCheckRequest.GetRequestMetadata(), nil
 }
 
-func validateCheckRequest(ctx context.Context, req *openfgav1.CheckRequest, typesys *typesystem.TypeSystem) error {
-	if !validator.RequestIsValidatedFromContext(ctx) {
-		if err := req.Validate(); err != nil {
-			return status.Error(codes.InvalidArgument, err.Error())
-		}
-	}
-
```

Comment:
this request-level validation is happening in `server.go` now

Reply:
**Question:** Does this validation still get applied if openfga is consumed as a library as opposed to as an API?

Reply:
good question, it does. Running as a library is done [like this](https://pkg.go.dev/github.com/openfga/openfga/pkg/server#example-NewServerWithOpts), which means you will get the request-level validation as part of the server handler